### PR TITLE
Add checkboxes to exclude specific result (true, false, undefined) from the list of checks

### PR DESCRIPTION
**Pull request by Aberdeener**
Screen Shot 2021-12-24 at 11 02 49 PM

### CODE REVIEW

**Review by lucko — APPROVED** [review]

File: src/views/Verbose.vue [hunk 1]
```diff
@@ -71,6 +71,41 @@
             v-model="filter"
             :placeholder="$t('verbose.filterPlaceholder')"
           >
+          <table>
```
Comment (Turbotailz) [inline src/views/Verbose.vue:74]: Can we please not use `table` here? Div tags should be good enough.

File: src/views/Verbose.vue [hunk 2]
```diff
@@ -71,6 +71,41 @@
...
+            <tr>
+              <td>
+                <div
+                  :class="{ 'exclude-result': true, 'selected': isExcluded('true') }"
```
Comment (Turbotailz) [inline src/views/Verbose.vue:78]: It would be good to use actual checkbox input elements here

File: src/views/Verbose.vue [hunk 3]
```diff
@@ -71,6 +71,41 @@
...
+                  @click="excludeResult('true')"
+                >
+                  <span></span>
+                </div>
+              </td>
+              <td>{{ $t('verbose.exclude') }} <code :class="'true'">true</code></td>
```
Comment (Turbotailz) [inline src/views/Verbose.vue:84]: When attributes are strings, you don't need to bind them like `:class="'true'"`, you can just write `class="true"`

File: src/views/Verbose.vue [hunk 4]
```diff
@@ -71,6 +71,41 @@
...
+            </tr>
+            <tr>
+              <td>
+                <div
+                  :class="{ 'exclude-result': true, 'selected': isExcluded('false') }"
```
Comment (Turbotailz) [inline src/views/Verbose.vue:89]: This class can be written like

`:class="['exclude-result', { selected: isExcluded('false') }]"`

**Review by Turbotailz — CHANGES_REQUESTED** [review]
Looks good, just a few things <3

File: src/views/Verbose.vue (hunk 3 above)
Comment (Aberdeener) [reply src/views/Verbose.vue:84]: Not sure how I missed that, thanks 😄

File: src/views/Verbose.vue (hunk 2 above)
Comment (Aberdeener) [reply src/views/Verbose.vue]: There aren't any other `<input type="checkbox"...` in the codebase, which is why I used the same code from the Node.vue file

Comment (Turbotailz) [reply src/views/Verbose.vue]: Then I am an idiot and you shouldn't follow my every step 🤪 
It's not very accessible, that's all.

File: src/views/Verbose.vue [hunk 5]
```diff
@@ -71,6 +71,37 @@
             v-model="filter"
             :placeholder="$t('verbose.filterPlaceholder')"
           >
+          <div>
+            <div
```
Comment (Turbotailz) [inline src/views/Verbose.vue:75]: This should be a loop actually.

```
<div 
  v-for="value in ['true', 'false', 'undefined']"
  :class="['exclude-result', { selected: isExcluded(value) }]"
  @click="excludeResult(value)"
>
... etc
```

Comment (Turbotailz) [reply src/views/Verbose.vue:75]: Then you can remove the repeated ones

**Review by Turbotailz — CHANGES_REQUESTED** [review]
Thanks for updates, just one last thing <3

File: src/views/Verbose.vue (hunk 5 above)
Comment (Aberdeener) [reply src/views/Verbose.vue:75]: That is unbelievably sexy, thank you for the advice

File: src/views/Verbose.vue [hunk 6]
```diff
@@ -71,6 +71,16 @@
...
             :placeholder="$t('verbose.filterPlaceholder')"
           >
+          <div
+            v-for="value in ['true', 'false', 'undefined']" :key="value"
+            :class="['exclude-result', { selected: isExcluded(value) }]"
+            @click="excludeResult(value)"
+          >
+            <span></span>
+            <p>
+              {{ $t('verbose.exclude') }} <code :class="value">{{ value }}</code>
```
Comment (Turbotailz) [inline src/views/Verbose.vue:81]: Please add language definitions for `verbose.exclude`
image

**Review by Turbotailz — CHANGES_REQUESTED** [review]

**Review by Turbotailz — APPROVED** [review]

**Pull request merged**